README.md

```diff
@@ -29,7 +29,7 @@ On Windows, you can use directory junctions instead of symlinks (symlinks requir
 ```powershell
 # Using PowerShell
 PS> git clone https://github.com/sublimehq/Packages.git
-PS> cmd /c mklink /J "$env:APPDATA/Sublime Text 3/Packages/Python" (convert-path ./Packages/Python)
+PS> cmd /c mklink /J "$env:APPDATA/Sublime Text/Packages/Python" (convert-path ./Packages/Python)
```

Comment:
there were some concerns about this change in a previous PR:
https://github.com/sublimehq/Packages/pull/2686#discussion_r556759936

Reply:
I've reverted those changes. That's a dilemma. :(